config/core/configmaps/gc.yaml

```diff
@@ -47,10 +47,35 @@ data:
     # resync period (10 hours).
     stale-revision-timeout: "15h"
 
-    # Minimum number of generations of revisions to keep before considering
-    # them for GC
+    # Minimum number of generations of non-active revisions to keep before
+    # considering them for GC.
     stale-revision-minimum-generations: "20"
 
     # To avoid constant updates, we allow an existing annotation to be stale by this
     # amount before we update the timestamp.
     stale-revision-lastpinned-debounce: "5h"
+
+
+    # ---------------------------------------
+    # V2 Garbage Collector Settings
+    # ---------------------------------------
+    #
+    # These settings are enabled via the 'responsive-revision-gc' feature flag.
+    # ALPHA NOTE: This feature is still experimental and under active development.
+
+    # Duration since revision creation before considering it for GC.
+    gc-retain-since-create-time: "48h"
+
+    # Duration since a route has pointed at the revision
+    # before considering it for GC.
+    gc-retain-since-last-active-time: "15h"
+
+    # Minimum number of non-active revisions to retain.
+    gc-min-stale-revisions: 20
+
+    # Maximum number of non-active revisions to retain regardless of
+    # creation or staleness time-bounds.
```

Comment:
So the maximum total number of revisions for the service is going to be this number + all the active ones? Worth mentioning?

Reply:
That's correct. More directly, that's the total number per-config (although we only have one config per service) since we reconcile GC on each configuration.

Reply:
added on to the yaml comment